VimeoUpload/Upload/Descriptor System/Cameo/CAMUploadReqest.swift

```diff
@@ -12,15 +12,13 @@ public enum CAMUploadRequest: String
 {
     case CreateVideo
     case UploadVideo
-    case ActivateVideo
-    case VideoSettings
     case CreateThumbnail
     case UploadThumbnail
     case ActivateThumbnail
     
     static func orderedRequests() -> [CAMUploadRequest]
     {
-        return [.CreateVideo, .UploadVideo, .ActivateVideo, .VideoSettings, .CreateThumbnail, .UploadThumbnail, .ActivateThumbnail]
+        return [.CreateVideo, .UploadVideo, .CreateThumbnail, .UploadThumbnail, .ActivateThumbnail]
```

Comment:
I was under the impression that there was a 4th thumbnail request necessary, a `DELETE` call, maybe not?